test/module/evaluators.js

```diff
@@ -127,7 +127,7 @@ test('createSafeEvaluatorWhichTakesEndowments - options.transforms', t => {
 });
 
 test('createSafeEvaluatorWhichTakesEndowments', t => {
-  t.plan(9);
```

Comment:
Good that you're maintaining the existing convention of this repository. But I'll note that Agoric should ban `t.plan(x)` everywhere and switch all of them to `t.end()`.

Reply:
I've created an issue to track this change (#27).